src/plugins/theme/screens.js

```diff
@@ -3,7 +3,6 @@
  */
 module.exports = function () {
   return {
-    xs: '0',
```

Comment:
I decided to leave out the xs = 0 breakpoint to follow TW convention. And this also cuts down the file size by a whole MB.
https://tailwindcss.com/docs/breakpoints

I also adjusted our custom classes `.su-centered-container` and `.su-grid-gap` (which loops over all breakpoint) to work with this change.